project/Common.scala

```diff
@@ -5,9 +5,8 @@ object Common {
   val settings: Seq[Def.Setting[_]] = Seq(
     scalaVersion := "2.12.6",
     organization := "uk.ac.wellcome",
-
-    resolvers ++= Seq("S3 releases" at "s3://releases.mvn-repo.wellcomecollection.org/"),
-
+    resolvers ++= Seq(
+      "S3 releases" at "s3://releases.mvn-repo.wellcomecollection.org/"),
```

Comment:
nit: I know this is probably just your IDE having opinions, but I like to keep this sort of unrelated change out of PRs to keep the diffs tight and easy to review.

Reply:
Same, but it's should have been picked up by scalafmt anyway, so I am going to leave it.